api/snapshot.go

```diff
@@ -182,6 +182,8 @@ func (s *Server) SnapshotBackup(w http.ResponseWriter, req *http.Request) (err e
 		labels[types.KubernetesStatusLabel] = string(kubeStatus)
 	}
 
+	labels[types.LonghornVolumeAccessModeLabel] = string(vol.Spec.AccessMode)
```

Comment:
Why not use with [LonghornLabelKeyPrefix](https://github.com/longhorn/longhorn-manager/blob/master/types/types.go#L274-L276)?

Reply:
Thanks, I will follow the suggestion. 
https://github.com/longhorn/longhorn-manager/blob/0d4593a5d1c666a1d1c8f6afeadffd2bde4a3c9f/webhook/resources/backup/mutator.go#L71

And the label name will be `longhorn.io/volume-access-mode` 
![image](https://user-images.githubusercontent.com/17548901/188202877-fcda0b37-4126-4b64-80d2-aabd645fba7c.png)